docs/snippets/gettingStarted/webApi/simpleApi.ts

```diff
@@ -139,6 +141,40 @@ export const shoppingCartApi =
         return NoContent();
       }),
     );
+
+    // Get Shopping Cart
+    router.get(
+      '/clients/:clientId/shopping-carts/current',
+      on(async (request: GetShoppingCartRequest) => {
+        const shoppingCartId = getShoppingCartId(
+          assertNotEmptyString(request.params.clientId),
+        );
+
+        const result = await eventStore.aggregateStream<
+          ShoppingCart,
+          ShoppingCartEvent
+        >(shoppingCartId, {
+          evolve,
+          getInitialState,
+        });
+
+        const productItems: ProductItem[] = Array.from(
+          result.state.productItems,
+        ).map(([productId, quantity]) => ({
+          productId,
+          quantity,
+        }));
+
+        return OK({
+          body: {
+            clientId: assertNotEmptyString(request.params.clientId),
+            id: shoppingCartId,
+            productItems,
+            status: result?.state.status,
```

Comment:
**SUGGESTION:** I think that instead of using `?` here, you could return `NotFound` status if result is `null`